internal/test/scorecard/common_utils.go

```diff
@@ -497,3 +597,37 @@ func cleanupCryostat(r *scapiv1alpha3.TestResult, client *CryostatClientset, nam
 		r.Log += fmt.Sprintf("failed to delete Cryostat: %s\n", err.Error())
 	}
 }
+
+func cleanupClusterCryostat(r *scapiv1alpha3.TestResult, client *CryostatClientset, name string, namespace string) {
+	cr := &operatorv1beta1.ClusterCryostat{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: name,
+		},
+		Spec: operatorv1beta1.ClusterCryostatSpec{
+			InstallNamespace: namespace,
+		},
+	}
+
+	r.Log += "trying to delete ClusterCryostat"
```

Comment:
Nit: This is for testing purpose right? We can remove it?

Reply:
oh yes lol I forgot to remove all the testing stuff